CPAC/func_preproc/func_preproc.py

```diff
@@ -977,7 +977,7 @@ def get_motion_ref(wf, cfg, strat_pool, pipe_num, opt=None):
     if opt != 'mean' and opt != 'median' and opt != 'selected_volume':
```

Comment:
I was so confused why it would matter if that key was `selected volume` or `selected_volume`, but I see it's this line :point_up: